src/components/options/providers/add.svelte

```diff
@@ -74,6 +75,10 @@ function validateLabel() {
 
 function validateLink() {
   updateError("link", !isUrl(editData.link));
+
+  warning = !isSearchable
+    ? "The link contains neither TESTSEARCH nor TESTB64SEARCH"
+    : null;
```

Comment:
The warning is never show, probably because you should be doing `warning = !isSearchable(editData.link)`